Project.toml

```
@@ -22,7 +22,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 XMLDict = "228000da-037f-5747-90a9-8195ccbf91a5"
 
 [compat]
-Compat = "3.29"
+Compat = "4.2"
```

Would be nice to not remove compat:
```suggestion
Compat = "3.29, 4.2"
```
(But I think for testing if it works, the tighter bound is OK so we know it ran with it).

---

Also I probably should have put `4` rather than `4.2` since if it works with `3.29` it's extremely likely to work with `4.0`.  Have changed this so it's more generous.